site/docs/use/00-user.md

```diff
@@ -6,41 +6,194 @@ title: User Manual
 
 > All documentations are under active work and subject to change soon!
 
-### Self Register User
+### Organization and user registration
```

Comment:
With the new docs structure, we should split this file into multiple for each part and add it to the sidenav